src/web3client/client.js

```diff
@@ -227,11 +231,17 @@ class Client {
   }
 
   getAccountAddress () {
-    return this.agentAccounts.address;
+    if(this.agentAccounts !== undefined) {
+      return this.agentAccounts.address;
+    }
+    return;
```

Comment:
afaik this could be simplified to just `return this.agentAccounts.address;` as a `return` without value defines `undefined` anyway.